tests/schemas/invalid/test_invalid_list_collector_with_routing.json

```diff
@@ -0,0 +1,156 @@
+{
+  "mime_type": "application/json/ons/eq",
+  "schema_version": "0.0.1",
+  "data_version": "0.0.2",
+  "survey_id": "0",
+  "title": "Test ListCollector",
+  "theme": "default",
+  "description": "A questionnaire to test ListCollector",
+  "metadata": [
+    {
+      "name": "user_id",
+      "validator": "string"
+    },
+    {
+      "name": "period_id",
+      "validator": "string"
+    },
+    {
+      "name": "ru_name",
+      "validator": "string"
+    }
+  ],
+  "sections": [
+    {
+      "id": "section",
+      "groups": [
+        {
+          "id": "group",
+          "title": "List",
+          "blocks": [
+            {
+              "id": "list-collector",
+              "type": "ListCollector",
+              "populates_list": "people",
+              "question": {
+                "id": "confirmation-question",
+                "type": "ListCollectorConfirmation",
+                "title": "Does anyone else live here?",
+                "answers": [
+                  {
+                    "id": "anyone-else",
+                    "mandatory": true,
+                    "type": "ListCollectorConfirmationRadio",
+                    "options": [
+                      {
+                        "label": "Yes"
```

Comment:
I think that relying on an empty value do determine the add action is unclear. It's also not ideal to have new types for the block, question and answer. I've discussed this with @jonnyshaw89 and we now prefer a new block level property of `add_answer_value` -  string value that matches whatever radio answer should be directed to the add block. This would also allow us to remove the special question and answer types.

Reply:
Addressed